.github/workflows/build.yml

```diff
@@ -40,6 +40,8 @@ jobs:
       with:
         platforms: all
     - uses: pypa/cibuildwheel@v2.21.2
+      env:
+      CXX: "g++-12"
```

Comment:
```suggestion
        CXX: "g++-12"
```

Reply:
Yeah, tried that! 4753d22e2cc1a20951398424b7d84e3ac639bd74